### ISSUE TITLE
Issues with running video colorizer script in JupyterLab

### ISSUE THREAD

**Issue by TheVincinator** [jantic/DeOldify#307]
Hello, I am having a problem with running DeOldify video colorization on JupyterLab. It gives me an error when I run the colorizer line. Is there any way of fixing this issue?

![Jupyter Lab DeOldify Issue](https://user-images.githubusercontent.com/68995370/104787604-e9bd3b80-575d-11eb-9899-52865271898e.png)

**Comment by TheVincinator** [comment]
![Jupyter Lab DeOldify Issue part  2](https://user-images.githubusercontent.com/68995370/104829839-e9986b00-5845-11eb-8183-d25ca8c9e860.png)

I was also having trouble with the other video scripts.

**Comment by jantic** [comment]
@TheVincinator You have a few things possibly going on there:

1. The directory you install deoldify to needs recursive read/write permissions open for the program.  And you need the models directory created and populated with models. See [here](https://github.com/jantic/DeOldify/issues/250 ).
2. The errors you are getting running it are suggesting that you either don't have everything installed or forgot to activate the conda environment. 

Bottom line:  The readme has instructions covering all this.  The conda install route is the way to go to keep your sanity.
